drift/lib/isolate.dart

```diff
@@ -39,13 +39,17 @@ class DriftIsolate {
   /// can reconstruct a [DriftIsolate] by using [DriftIsolate.fromConnectPort].
   final SendPort connectPort;
 
+  /// The flag indicating whether messages between this [DriftIsolate]
+  /// and the [DriftServer] should be serialized.
+  final bool serialize;
+
   /// Creates a [DriftIsolate] talking to another isolate by using the
   /// [connectPort].
-  DriftIsolate.fromConnectPort(this.connectPort);
+  DriftIsolate.fromConnectPort(this.connectPort, {this.serialize = true});
```

Comment:
Could you also add the `{@macro drift_isolate_serialize}` macro to the docs on this constructor?

Reply:
I moved the template here and removed it from `connect` and `shutdown`